tool/check-code.sh

```diff
@@ -87,86 +84,62 @@ function check_formatting() {
   fi
 }
 
+# Extract snippets, analyze and check formatting
+# Null safety flag set
+EXAMPLE_ROOT="examples"
+echo "=> ANALYZING and testing apps in $EXAMPLE_ROOT/* --null-safety"
```

Comment:
`--null-safety` in this line should be removed since it's removed from the test?